qwerty/eo-qwerty.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+This keyboard adds the Esperanto-specific diacritic letters: ĉ ĝ ĥ ĵ ŝ ŭ as northwest swipes.
+It also adds the (obsolete) Spesmilo currency sign and the culturally significant star symbol.
+Contributed by Haley Halcyon
+-->
+<keyboard name="QWERTY (Esperanto)" script="latin">
+  <row>
+    <key c="q" ne="1" se="loc esc"/>
+    <key c="w" nw="~" ne="2" sw="\@"/>
+    <key c="e" nw="!" ne="3" sw="\#" se="loc €"/>
+    <key c="r" ne="4" sw="$"/>
+    <key c="t" ne="5" sw="%"/>
+    <key c="y" ne="6" sw="^"/>
+    <key c="u" nw="ŭ" ne="7" sw="&amp;"/>
+    <key c="i" ne="8" sw="*"/>
+    <key c="o" ne="9" sw="(" se=")"/>
+    <key c="p" ne="0"/>
+  </row>
+  <row>
+    <key shift="0.5" c="a" nw="loc tab" ne="`"/>
+    <key c="s" ne="loc §" nw="ŝ" se="₷" sw="loc ß"/>
+    <key c="d" ne="★" nw="☆" />
```

Comment:
You could make one star be the [shift](https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/compose/shift.json) version of the other by modifying the app.